README.md

```diff
@@ -331,4 +344,11 @@ The `outbound-edi` function can be invoked via the UI for testing.
 
   </details>
 
-5. You can view the file using the [Buckets Web View](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
\ No newline at end of file
+5. You can view the file using the [Buckets UI](https://www.stedi.com/app/buckets). As shown above, the output of the function includes the `bucketName` and `key` (path within the bucket) of where the generated EDI was saved.
+
+# Customizing the Bootstrap Workflow
+
+The bootstrap workflow uses a sample trading partner to set up and test the read and write EDI workflows. You can customize the bootstrap workflow for your use case by doing one or all of the following:
+- [Add a partner profile](https://www.stedi.com/docs/bootstrap/adjusting-the-workflow#add-a-trading-partner-profile) to replace the test trading partner with your real trading partners' details and requirements.
```

Comment:
nit: there's a contradiction here in `Add` and `replace` (could either be `Edit` and `replace`, or just `Add` and maybe `to represent your real trading...)`